media/iso_iec_base_media.ksy

```diff
@@ -1,8 +1,19 @@
 meta:
-  id: quicktime_mov
+  id: iso_iec_base_media
+  file-extension:
+    # the list of extensions is according to https://en.wikipedia.org/wiki/ISO/IEC_base_media_file_format
+    - mp4
+    - 3gp
+    - 3g2
+    - mj2
+    - dvb
+    - dcf
+    - m21
+    - f4v
   application: QuickTime, MP4 ISO 14496-14 media
```

Comment:
I guess it should be either an array, or removed at all.

Reply:
What? The `file-extension` list? I think it's definitely good to keep it here, otherwise the people which extensions this format is associated with. And in the future, we may add some search functionality to the Formats Gallery. Most people won't search for `iso_iec_base_media`, they'll search `mp4` or `3gp`.

And `it is` an array, just written in the multiline format. But I think it's not wrong at all, it's clearly organized and it plays well with the VCS too. And in KSY we generally prefer _Block (multiline) YAML style_ to single line syntax.

Reply:
`application`